src/AdvApi32.Desktop/AdvApi32.Helpers.cs

```diff
@@ -40,7 +40,7 @@ public static unsafe IEnumerable<ENUM_SERVICE_STATUS> EnumServicesStatus()
                     scmHandle,
                     ServiceType.SERVICE_WIN32,
                     ServiceStateQuery.SERVICE_STATE_ALL,
-                    null,
+                    IntPtr.Zero,
```

Comment:
This change was necessary because there are two overloads that accept `null` for this parameter (`byte[]` and `byte*`). So either we'd have to cast null to one of these, or just use `IntPtr.Zero`.